src/merge_stop_areas.rs

```diff
@@ -211,6 +212,13 @@ fn apply_rules(
                 }
             }
         }
+        for ticket in &mut ticket_use_perimeters_updated {
+            if ticket.object_type == ObjectType::StopArea
```

Comment:
As seen together, there is no StopArea in TicketUsePerimeter (just Line and Network).
But it's on TicketUseRestriction (RestrictionType OD)